git_machete/cli.py

```diff
@@ -631,14 +631,14 @@ def launch(orig_args: List[str]) -> None:
             machete_client.read_definition_file()
             res = []
             if category == "addable":
-                def strip_first_fragment(remote_branch: str) -> str:
-                    return re.sub("^[^/]+/", "", remote_branch)
+                def strip_first_fragment(remote_branch: RemoteBranch) -> LocalBranch:
+                    return LocalBranch.of(re.sub("^[^/]+/", "", remote_branch))
 
                 remote_counterparts_of_local_branches = utils.map_truthy_only(
-                    lambda _branch: git.get_combined_counterpart_for_fetching_of_branch(_branch),
+                    lambda _branch: LocalBranch.of(git.get_combined_counterpart_for_fetching_of_branch(_branch)),
                     git.get_local_branches())
-                qualifying_remote_branches = excluding(git.get_remote_branches(),
-                                                       remote_counterparts_of_local_branches)
+                qualifying_remote_branches: List[RemoteBranch] = excluding(git.get_remote_branches(),
+                                                                           [RemoteBranch.of(b) for b in remote_counterparts_of_local_branches])
```

Comment:
```suggestion
                                                                           {RemoteBranch.of(b) for b in remote_counterparts_of_local_branches})
```
Maybe `set()` instead of `list()` in here? This way we don't have to iterate through whole list with each remote to be compared and we're avoiding checking multiple times `None` that can pop in the list